db/versions/0013.yml

```
@@ -0,0 +1,166 @@
+version: 13
+migrationScript: 0013-migration.sql
+downgradeScript: 0013-downgrade.sql
+methods:
+  get_worker_2:
+    description: Get an existig worker. The returned table will have one or (if no such worker is defined) zero rows.
```

Lol, this typo is multiplying quickly :)

```suggestion
    description: Get an existing worker. The returned table will have one or (if no such worker is defined) zero rows.
```